codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/rustlang/Writable.kt

```diff
@@ -18,14 +18,34 @@ typealias Writable = RustWriter.() -> Unit
  */
 fun writable(w: Writable): Writable = w
 
-fun writable(w: String): Writable = writable { rust(w) }
+fun writable(w: String): Writable = writable { writeInline(w) }
```

Comment:
This is the most controversial change - it makes the asserts easier in the tests but might have knock-on effects.

Reply:
I doubt this method is used in a lot of places tbh, the codegen diff should show any changes

Reply:
I added a similar templating fn (`rustInlineTemplate`) in my recent PR for the same reason (easy asserts)

Reply:
@Velfi shall I undo this and use that in my asserts instead?